src/Components/AddContacts/AddContact.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
```

Comment:
make attention to file names.

_src/Components/AddContacts/index.js_ is better name